test/autorest/errorsgroup/errorsgroup_test.go

```diff
@@ -104,7 +105,12 @@ func TestGetPetByIDSuccess1(t *testing.T) {
 }
 
 func TestGetPetByIDSuccess2(t *testing.T) {
-	t.Skip("multiple responses with different schemas NYI")
+	client := getClient(t)
+	result, err := client.GetPetByID(context.Background(), "django")
```

Comment:
If this is a Get should there also be a comparison of what we actually got? (If we are receiving something)

Reply:
The response for this test only returns a 200 (no response body or headers).

Reply:
Wait let me look into this a bit more....

Reply:
Per the swagger, for an HTTP 200 response you get back a Pet schema.  For an HTTP 202 response there is no schema.

Reply:
Ah ok, that sounds good then! Is the 202 the final state though for this test?

Reply:
Well yeah it should be, nvm. Looks good!